documentation/doxygen/filter.cxx

```diff
@@ -194,7 +195,7 @@ void FilterClass()
             if (m) {
                fclose(m);
                m = 0;
-               ExecuteCommand(StringFormat("root -l -b -q \"makeimage.C(\\\"%s\\\",\\\"%s\\\",\\\"%s\\\",true,false)\""
+               ExecuteCommand(StringFormat("root -l -b -q \"makeimage.C+O(\\\"%s\\\",\\\"%s\\\",\\\"%s\\\",true,false)\""
                                               , StringFormat("%s_%3.3d.C", gClassName.c_str(), gMacroID).c_str()
```

Comment:
I do not understand this +0 you added

Reply:
Exactly. This makes a big difference for tutorials like RDataframe.